datasift-writer/src/main/resources/writer-example.json

```diff
@@ -13,7 +13,8 @@
     "username": "",
     "api_key": "",
     "source_id": "",
-    "bulk-size": 1000,
+    "bulk-size": 100000,
```

Comment:
The bulk-size property has a typo. I believe it should be bulk_size so that the value is read correctly, it happens to be the same as the default value anyway. Is it worth adding a test to see if it works correctly with a different value?